components/ScannerUI.vue

```diff
@@ -1,5 +1,15 @@
 <template>
   <div class="fixed-full bg-black">
+    <!-- Video Stream -->
+    <div
+      id="video-container"
```

Comment:
Avoid `id` generally with reusable components, as an application could opt to put multiple components of the same kind on the same page. That being said, no need to change this right now, but it is something to consider when we refactor some of this scanner code into a reusable vanilla (non-framework-specific) JS API.